contracts/LifCrowdsale.sol

```diff
@@ -126,10 +137,17 @@ contract LifCrowdsale is Ownable, Pausable {
     TokenPurchase(msg.sender, beneficiary, weiAmount, tokens);
   }
 
-  function addPresaleTokens(address beneficiary, uint256 tokens) onlyOwner {
+  function addPrivatePresaleTokens(address beneficiary, uint256 totalWei) onlyOwner {
     require(block.number < startBlock);
     require(beneficiary != address(0));
-    require(tokens > 0);
+    require(totalWei > 0);
+
+    uint256 tokens = totalWei.mul(privatePresaleRate);
+
+    uint256 presaleWei = totalPresaleWei;
```

Comment:
why a separate `presaleWei` variable if we can just use `totalPresaleWei` (or whatever new name it gets)?